src/index.css

```
@@ -16,3 +18,13 @@ body {
   background-color: var(--color-primary);
   overflow-x: hidden;
 }
+
+@utility flex-center {
```

Stylelint (17.3.0)

[error] Unexpected unknown at-rule "@utility" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)